twisted/protocols/test/test_tls.py

```diff
@@ -742,6 +743,14 @@ def cbConnectionDone(ignored):
         connectionDeferred.addCallback(cbConnectionDone)
         return connectionDeferred
 
+    def test_hugeWrite_TLSv1(self):
+        return self.hugeWrite()
+
+    def test_hugeWrite_TLSv1_1(self):
+        return self.hugeWrite(method=TLSv1_1_METHOD)
+
+    def test_hugeWrite_TLSv1_(self):
```

Comment:
You're right, sorry. I've updated this.

On 27 Jul 2016 1:43 pm, "Craig Rodrigues" <notifications@github.com> wrote:

> In twisted/protocols/test/test_tls.py
> <https://github.com/twisted/twisted/pull/424#discussion_r72376695>:
>
> > @@ -742,6 +743,14 @@ def cbConnectionDone(ignored):
> >          connectionDeferred.addCallback(cbConnectionDone)
> >          return connectionDeferred
> >
> > +    def test_hugeWrite_TLSv1(self):
> > +        return self.hugeWrite()
> > +
> > +    def test_hugeWrite_TLSv1_1(self):
> > +        return self.hugeWrite(method=TLSv1_1_METHOD)
> > +
> > +    def test_hugeWrite_TLSv1_(self):
>
> Should this method be named test_hugeWrite_TLSv1_2 () ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/twisted/twisted/pull/424/files/7784c1736dc00d99e37c77cab057fc8d4ecf77e4#r72376695>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGzhD3dQyxp_oQw8xNAP-aOP0qR4gSHMks5qZtPsgaJpZM4JVvea>
> .
>